test/boost/sstable_compaction_test.cc

```diff
@@ -3866,17 +3866,14 @@ SEASTAR_TEST_CASE(test_offstrategy_sstable_compaction) {
             auto mut = mutation(s, pk);
             ss.add_row(mut, ss.make_ckey(0), "val");
 
-            auto cf = env.make_table_for_tests(s, tmp.path().string());
+            auto cf = env.make_table_for_tests(s, tmp.path().string(), version);
```

Comment:
This caught my ~~eagle~~eye . Why is version argument added here? Is it runaway from one of previous patches?

Reply:
It's needed for cf.make_sstable to use that version.
I'm refactoring the patches in finer resolution in the next version so it will be more visible and better explained